server/services/datacommons.py

```diff
@@ -336,9 +336,13 @@ def resolve(nodes, prop):
   return post(url, {'nodes': nodes, 'property': prop})
 
 
-def nl_search_vars(queries, index_type, skip_topics=False, reranker=''):
+def nl_search_vars(queries,
+                   index_types: List[str],
+                   skip_topics=False,
+                   reranker=''):
   """Search sv from NL server."""
-  url = f'{current_app.config["NL_ROOT"]}/api/search_vars?idx={index_type}'
+  idx_params = ','.join(index_types)
```

Comment:
Oh I see, so https://github.com/datacommonsorg/website/blob/8d67e1c1592d36cc796c93ed67b75abbbed2e37d/nl_server/routes.py#L92 is already supported.